src/components/DpInsertableRecommendation/DpInsertableRecommendation.vue

```diff
@@ -0,0 +1,151 @@
+<template>
```

Comment:
Well... Using this component across two repositories does not make it a generic component at all. If we further go this way we'll make demosplan-ui the discount bin of frontend components, which is quite the opposite of what it should be in terms of usability and focus. 

In my opinion it would be ok to duplicate that component, if we need it in two places.

Reply:
Allright, done here: https://github.com/demos-europe/demosplan-ui/pull/473/commits/8bfaee7960c1cdc19020cafd3470dc7e94e376c8